custom/icds_reports/ucr/data_sources/child_cases_monthly_v2.json

```diff
@@ -407,6 +407,17 @@
         "type": "boolean",
         "column_id": "child_count_male"
       },
+     {
+      "filter": {
+      "type": "named",
+      "name": "is_resident",
+      "comment": null
```

Comment:
If its not necessary to put `"comment": null`, better to just avoid it since its handled by default. Just blurs the other essential things.

Reply:
have removed comment: null